packages/backend-common/src/s3.ts

```diff
@@ -60,15 +60,21 @@ export const getSignedDownloadUrl = async (
 	bucket: string,
 	key: string,
 	expiresIn: number,
-) =>
-	await getSignedUrlSdk(
+	overrideFilename?: string,
+) => {
+	const responseContentDisposition = overrideFilename
+		? `attachment; filename="${overrideFilename}"`
```

Comment:
It might be worth sanitising this string since its provided by the user? 
I suppose because of file access controls, users could only do a header injection attack on themselves or the dev who downloads the file, but might be worth locking down anyway

Reply:
yes great shout, added in https://github.com/guardian/transcription-service/pull/135/commits/65d3a5547227512dbca67a7edacd0b03a79e1b34